testdata/public/package/multiple-false/0.0.1/index.json

```diff
@@ -19,8 +19,8 @@
     "/package/multiple-false/0.0.1/manifest.yml",
     "/package/multiple-false/0.0.1/docs/README.md",
     "/package/multiple-false/0.0.1/dataset/foo/manifest.yml",
-    "/package/multiple-false/0.0.1/dataset/foo/fields/stream.yml",
-    "/package/multiple-false/0.0.1/dataset/foo/agent/stream/stream.yml",
+    "/package/multiple-false/0.0.1/dataset/foo/fields/stream.yml.hbs",
```

Comment:
Any chance we could take the `.hbs` part into a separate PR. We might need here some discussions with Kibana team around this and also makes it easier to reference to it later on if questions come up.

+1 on this change BTW.

Reply:
Hmm... honestly I would keep it here as I already applied in multiple places (simpler option at the moment, preserving we will rename to this). Also it's a parsed template, so definetely a .hbs file. WDYT?

Reply:
Ok, in this case we should add a line to the changelog. And perhaps we can do a Github issue instead to describe the modifivation of the change instead and reference this PR.

Reply:
Changelog entry added.